### DISCUSSION TITLE
How to handle headers with `-` in the name.

### DISCUSSION THREAD

**Discussion by ChillFish8** [poem-web/poem#208] [Q&A] [answered]
Currently, you can get a header by doing something along the lines of

`connection: Header<String>` with poem openapi

but how can you get a header which has a `-` in the name, doing `foo_bar` wont match the headers `foo-bar`

**Comment by sunli829** [comment, marked as answer]
You can rename parameter names. 😁

```rust
async fn a(&self, #[oai(name="foo-bar")] foo_bar:Query<i32>) {
}
```

**Reply by ChillFish8** [reply]
Completely forgot this existed 😅
